src/utils.py

```diff
@@ -32,23 +32,17 @@ def __init__(self, execution_point: ServiceKey, url: str = None):
 
 
 def encode_for_sparql(string: str):
-    return string \
-        .replace(" ", "%20") \
-        .replace("<", "%3C") \
-        .replace(">", "%3E") \
-        .replace("(", "%28") \
-        .replace(")", "%29")
-
-
-def encode_for_fdk_base_sparql(string: str):
-    return string \
+    trim_str = string.strip()
+    return trim_str \
         .replace(" ", "%20") \
         .replace("<", "%3C") \
         .replace(">", "%3E") \
         .replace("(", "%28") \
         .replace(")", "%29") \
         .replace("{", "%7B") \
-        .replace("}", "%7D")
+        .replace("}", "%7D") \
+        .replace("'", "%27") \
+        .replace("*", "%2A").replace("\\", "\\\\")
```

Comment:
Kunne vi brukt en bibliotek for encoding (f.eks. https://www.urlencoder.io/python/)?

Reply:
Dunno...
@rannveigskjerve Er det en grunn til at vi ikke har brukt et bibliotek?